test/controllers/systems_controller_test.rb

```diff
@@ -5,20 +5,17 @@
 class SystemsControllerTest < ActionDispatch::IntegrationTest
   setup do
     SystemsController.any_instance.expects(:authenticate_user)
+    SystemsController.any_instance.expects(:policy_scope).with(Host)
```

Comment:
This isn't going to work here once we add tests for `show`. You can put it in a method, though, if you want it more DRY.

Reply:
Done, I've kept it separate for now